js/client-rooms.js

```diff
@@ -28,7 +40,7 @@
 				this.lastUpdate = new Date().getTime();
 			}
 			var prevPos = this.$el.scrollTop();
-			this.$('button[name=joinRoomPopup]').focus();
+			// this.$('button[name=joinRoomPopup]').focus();
```

Comment:
Why?

Reply:
This is important for keyboard shortcuts, too.

Reply:
Constantly re-focusing this button makes it impossible to use the dropdown